text-ui-test/runtest.bat

```diff
@@ -7,15 +7,15 @@ REM delete output from previous run
 del ACTUAL.TXT
 
 REM compile the code into the bin folder
-javac  -cp ..\src -Xlint:none -d ..\bin ..\src\main\java\Duke.java
+javac  -cp ..\src -Xlint:none -d ..\bin ..\src\main\java\Main.java ..\src\main\java\Deadline.java ..\src\main\java\Event.java ..\src\main\java\Task.java ..\src\main\java\Todo.java
 IF ERRORLEVEL 1 (
     echo ********** BUILD FAILURE **********
     exit /b 1
 )
 REM no error here, errorlevel == 0
 
 REM run the program, feed commands from input.txt file and redirect the output to the ACTUAL.TXT
-java -classpath ..\bin Duke < input.txt > ACTUAL.TXT
+java -Dfile.encoding=UTF-8 -classpath ..\bin Main Deadline Event Task Todo < input.txt > ACTUAL.TXT
```

Comment:
I like the use of UTF-8 encoding here since it addresses an obvious issue in the output files